frontend/lib/evictionfree/declaration-builder/confirmation.tsx

```diff
@@ -14,24 +17,24 @@ const NYC_311_CONTACT_LINK =
 const LIST_OF_ORGANIZING_GROUPS_URL =
   "https://d3n8a8pro7vhmx.cloudfront.net/righttocounselnyc/pages/1232/attachments/original/1590279936/List_of_Tenant_Organizing_Groups_Across_NY_State.pdf?1590279936";
 
+const H2_CLASSNAME = "title is-size-4 is-size-5-mobile is-spaced";
+
 const checkCircleSvg = require("../../svg/check-circle-solid.svg") as JSX.Element;
 
-const TitleWithSymbol = () => (
+const renderTitleWithCheckCircle = (title: string) => (
   <div className="media">
     <div className="media-left">
       <i className="has-text-info">{checkCircleSvg}</i>
     </div>
     <div className="media-content">
-      <h2 className="title is-size-4-mobile">
-        You've sent your hardship declaration
-      </h2>
+      <h1 className="title is-size-4-mobile">{title}</h1>
```

Comment:
FYI @sraby the `withHeading` prop on `<Page>` actually takes a function that takes a title string and returns JSX... it's useful for custom page titles that improve DRY, so i've applied that to the title rendering function here.

Reply:
I also changed the `<h2>` to a `<h1>` so the page's heading structure is more semantically coherent (useful for screen readers etc).  It doesn't seem to affect the actual appearance of the page in any way.